src/victory-util/events.js

```diff
@@ -92,6 +92,9 @@ export default {
 
       // returns all eventKeys to modify for a targeted childName
       const getKeys = (childName) => {
+        if (target === "parent") {
+          return "parent";
+        }
         if (eventReturn.eventKey === "all") {
           return baseProps[childName] ?
             without(Object.keys(baseProps[childName]), "parent") :
```

Comment:
NIT: we're using both `lodash.keys` and `Object.keys` in this file. idk which is faster, but we might want to go with just one